npm/qsharp/ux/scatterChart.tsx

```diff
@@ -105,7 +107,7 @@ export function ScatterChart(props: {
     const centerX = (pointRect.left + pointRect.right) / 2;
     const divRect = topDiv.getBoundingClientRect();
     tooltip.style.left = `${centerX - divRect.left - halfWidth}px`;
-    tooltip.style.top = `${centerY - divRect.top + 12}px`;
+    tooltip.style.top = `${centerY - divRect.top}px`; // + 12}px`;
```

Comment:
What was this change for?

Reply:
Apologies, forgot to revert. Was trying to find a way around the padding hack.